src/game/game.cpp

```diff
@@ -1870,9 +1875,13 @@ ReturnValue Game::internalMoveItem(Cylinder* fromCylinder, Cylinder* toCylinder,
 				return ret;
 			}
 
+			// Looting analyser
 			if (it.isCorpse && toCylinder->getContainer()->getTopParent() == player && item->getIsLootTrackeable()) {
 				player->sendLootStats(item, static_cast<uint8_t>(item->getItemCount()));
 			}
+
+			player->onSendContainer(toCylinder->getContainer());
```

Comment:
Visual removal or addition of things inside any container is already done on postAdd and postRemove functions of each class. Calling another visual update of the entire container may be pointless from having the old client update ([ProtocolGame::sendRemoveContainerItem](https://github.com/opentibiabr/canary/blob/c37ecd3f5681fdb81b54a66fb51ad059370ce39a/src/server/network/protocol/protocolgame.cpp#L6280)). Maybe this new approach of a container update can be done here [Container::onRemoveContainerItem](https://github.com/opentibiabr/canary/blob/c37ecd3f5681fdb81b54a66fb51ad059370ce39a/src/items/containers/container.cpp#L306C6-L306C38) and here [Container::onAddContainerItem](https://github.com/opentibiabr/canary/blob/c37ecd3f5681fdb81b54a66fb51ad059370ce39a/src/items/containers/container.cpp#L276), to avoid any duplication and unnecessary message sent to the client and the effor wasted on iterations and callbacks inside of the container lists. Using the new 'isStoreInboxFiltered()' can suits better to specify which kind of protocolgame function will be called.